### ISSUE TITLE
Suggestion: + change monospaced font size independantly 

### ISSUE THREAD

**Issue by tomByrer** [brave/brave-browser#7267]
## Description 
When changing fonts, I need to change the monospaced font size independently of the other font sizes (Chrome & Firefox allow this).  Reasoning; monospaced fonts, esp the default, tend to be both smaller height & thinner lines.

## Miscellaneous Information:
Immediate uninstall for me, since I wanted to use Brave for web development, & need to read code in <code> blocks in Github, blogs, SO, etc.  Sorry I didn't keep Brave around long enough for screenshots.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!

**Comment by idea-list** [comment]
@bsclifton yes, this is still a problem. If you go to Brave's font settings there still is only a single font size slider. And that slider sets the size of every font to the specified value *except* for fixed-width fonts. For some reason you have chosen to hardcode fixed-width fonts to always be `-3` from the specified value. So if I set font size to be `16` then standard, serif, sans-serif and math fonts will all be set to `16` while monospaced fonts will be set to `13`. What I want is for all font sizes to be `16` in this case. Current behavior is very annoying and forces me to use extensions Stylus and make manual override for monospaced font sizes on every single website that respects user-configured browser defaults (which, ironically, are impossible to configure in Brave). Could you please reopen this issue (and hopefully fix it too)?
